main.py

```diff
@@ -406,6 +406,9 @@ def ui_access_wrapper(self, *args, **kwargs):
     get_redirect_route(
         r'/createhandler/imageupload/<exploration_id>',
         editor.ImageUploadHandler),
+    get_redirect_route(
+        r'/createhandler/editor_audioupload/<exploration_id>',
```

Comment:
@seanlip, any suggestion on this, like if we are going to use such URL for rte audio which is like `editor_audioupload` what do we expect to have for translators audio upload?

This is why I was thinking to have an asstesController which will handle a single URL for both the role with a decorator can_upload_audio. (I know this will not be helpful for future use as discussed. :)) 

Note: We are currently using  `/createhandler/audioupload/<exploration_id>` for translator audio upload.

Reply:
Initial sections that look like

/createhandler/editoraudioupload
/createhandler/translatoraudioupload

seems reasonable.